pyproject.toml

```diff
@@ -34,56 +33,48 @@ authors = [
   {name = "Jonathan Slavin", email = "jslavin@cfa.harvard.edu"},
   {name = "Nick Murpy", email="namurphy@cfa.harvard.edu"},
   {name = "Will Barnes"},
+  {name = "Nabil Freij"},
   {name = "Stuart Mumford"},
 ]
 
 dependencies = [
-  "astropy >= 5.1",
-  "cached-property >= 1.5.2",
+  "astropy >= 5.3.0",
   "matplotlib >= 3.5.0",
-  "numpy >= 1.24.0",
-  "requests >= 2.28.0",
+  "numpy>=1.23.5",
   "scikit-image >= 0.19.0",
-  "scipy >= 1.8.0",
-  # install setuptools to get pkg_resources
-  "setuptools; python_version >= '3.12'",
-  "sunpy[map] >= 4.0.0",
+  # !=1.10.0 due to https://github.com/scipy/scipy/issues/17718
+  "scipy>=1.9.0,!=1.10.0",
+  "sunpy[map] >= 5.0.0",
 ]
 
 [project.optional-dependencies]
 dev = [
+  "xrtpy[tests,docs]",
   "nox >= 2022.8.7",
-  "pre-commit >= 3.6.0",
 ]
 tests = [
   "pytest >= 8.0.0",
-  "pytest-allclose >= 1.0.0",
-  "pytest-xdist >= 3.6.1",
+  "pytest-astropy",
 ]
 docs = [
-  "docutils >= 0.19",
   "imageio >= 2.20.0",
   "ipykernel >= 6.20.0",
   "ipython >= 8.4.0",
   "ipywidgets >= 8.1.0",
-  "jinja2 != 3.1, >= 3.0.0",
   "nbconvert >= 7.7.0, < 7.14",
   "nbsphinx >= 0.9",
-  "numpydoc >= 1.5.0",
-  "pillow >= 9.1.0",
-  "pygments >= 2.12.0",
   "sphinx >= 7.3.0",
-  "sphinx-changelog >= 1.5.0",
+  "sphinx_automodapi >= 0.17.0",
   "sphinx-codeautolink >= 0.15.2",
   "sphinx-copybutton >= 0.5.2",
   "sphinx-gallery >= 0.16.0",
   "sphinx-hoverxref >= 1.4.0",
   "sphinx-issues >= 4.1.0",
-  "sphinx_automodapi >= 0.17.0",
-  "sphinx_rtd_theme >= 2.0.0",
+  "sphinx-rtd-theme >= 2.0.0",
+  # Need pkg_resources for sphinxcontrib-bibtex
+  # and this comes from setuptools
+  "setuptools",
```

Comment:
I don't think this is needed?

Reply:
This suppresses a warning that is raised in the build of the docs due to a dependency of bibtex not having been updated.

Reply:
Oh ok. Weird…